src/slices/adminApiSlice.ts

```
@@ -0,0 +1,29 @@
+import { apiSlice } from "./apiSlice";
+
+// Handle the response using the responseInterceptor
+const handleResponse = (response: any) => {
+    return response;
+};
+
+export const adminApiSlice = apiSlice.injectEndpoints({
+    endpoints: (builder) => ({
+
+        changePassword: builder.mutation({
+			query: ({ data }: { data: any }) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)